app/components/sample.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button, Empty, Card, Badge } from '@hero-design/react';
+
+// Snowflakes using styled-components;
+
+const customBtnStyles = {
+  padding: 30,
+};
+
+const badgeStyles = {
+  pt: 10,
+};
+
+const styles = {
+  btn: {
+    padding: 30,
+  },
+  badge: {
+    pt: 10,
+  },
+};
+
+const Sample = () => {
+  <>
+    // Snowflakes using classname
+    // Snowflakes using style prop
+    <Card style={{ width: 200 }} /> // acceptable
+    // Snowflakes using sx prop
+    <Badge sx={{ mt: 10 }} /> // acceptable
+    <Badge sx={{ pt: 20 }} />
```

Comment:
Snowflake detected! A component is customized via sx prop. Make sure to not use [prohibited CSS properties](https://docs.google.com/spreadsheets/d/1Dj8vqLdFaf-CSaSVoYqyYZIkGqF6OoyP7K4G1_9L62U/edit?usp=sharing).